config/api.yml

```diff
@@ -3046,6 +3046,15 @@
       :post:
       - :name: request_download
         :identifier: miq_report_view
+  :revokes:
```

Comment:
I find this weird...I would expect a revoke action on the /users/ endpoint.

Reply:
Either that or a revoke action on the /auth endpoint.

Reply:
yes `/user` sounds better to me and based on some api suprises with current way `/user` fits better to our api.

it is based on @abellotti [suggestion](https://github.com/ManageIQ/manageiq-api/issues/884#issuecomment-67668246), 

@abellotti it is ok If I will change it to `/user` endpoint ?

thanks